### PR TITLE
Update z-way docker to v4.1.3

### PR DESCRIPTION
**Pull request by msazanov**
In this update:
- the base image has been replaced
- redesigned the startup script
- z-way has been updated to version 4.1.3
- Remote access now works.
- added a special config.json for docker and HA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+3. Update the `docker-compose.yml` file with the correct device paths if necessary.
+
+4. Build and start the container:
+
+    ```sh
+    docker-compose build
+    docker-compose up
+    ```
+
+This server works only with controllers from Z-Wave.Me, such as RaZberry 2/5/7/Pro, mPCI module, UZB, and Z-Station.
```
Comment (PoltoS) [inline README.md]:
```suggestion
This server works only with controllers from Z-Wave.Me, such as RaZberry 2/5/7/Pro, mPCIe module, UZB, and Z-Station.
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
+    - **Windows** (using PowerShell):
+
+        ```powershell
+        Get-WmiObject Win32_SerialPort
+        ```
+
+    - **macOS**:
+
+        ```sh
+        ls /dev/tty.*
```
Comment (PoltoS) [inline README.md]:
```suggestion
        ls /dev/cua*
```

File: Dockerfile [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
-ENV TZ=Europe/Moscow
+# Block zbw key request
+RUN mkdir -p /etc/zbw/flags && touch /etc/zbw/flags/no_connection
 
-WORKDIR /opt
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+RUN apt-get update && \
+    apt-get install -qqy --no-install-recommends \
+    ca-certificates curl \
+    wget procps gpg iproute2 openssh-client openssh-server sudo logrotate jq
```
Comment (PoltoS) [inline Dockerfile]:
```suggestion
    wget procps gpg iproute2 openssh-client openssh-server sudo logrotate
```

Do we need jq?

File: README.md [hunk 4]
```diff
@@ -1,12 +1,39 @@
...
+        ```
+
+3. Update the `docker-compose.yml` file with the correct device paths if necessary.
+
+4. Build and start the container:
+
+    ```sh
+    docker-compose build
+    docker-compose up
```
Comment (PoltoS) [inline README.md]:
```suggestion
    docker compose build
    docker compose up
```

File: rootfs/opt/z-way-server/automation/defaultConfigs/config.json [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+            "publicAPI": false,
+            "createVDev": true,
+            "config": "configs/config",
+            "translations": "translations",
+            "ZDDX": "ZDDX"
+        },
+        "description": "Allows accessing ZigBee devices from attached ZigBee coordinator.\n(Added by default)"
+
+    }, {
+		"id": 3,
```
Comment (PoltoS) [inline rootfs/opt/z-way-server/automation/defaultConfigs/config.json]:
```suggestion
		"description": "Allows accessing Z-Wave devices from attached Z-Wave transceiver.\n(Added by default)"
	}, {
		"id": 2,
		"moduleId": "Zigbee",
		"active": false,
		"title": "Zigbee Network Access",
		"params": {
			"name": "zbee",
			"port": "/dev/ttyUSB1",
			"speed": 115200,
			"enableAPI": true,
			"publicAPI": false,
			"createVDev": true,
			"config": "configs/config",
			"translations": "translations",
			"ZDDX": "ZDDX"
		},
		"description": "Allows accessing ZigBee devices from attached ZigBee coordinator.\n(Added by default)"
	}, {
		"id": 3,
```

**Review by PoltoS — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (msazanov) [reply Dockerfile]: Right now not, but this good tool for future

File: README.md (hunk 4 above)
Comment (msazanov) [reply README.md]: Both variants works, but first is old )

File: Dockerfile (hunk 3 above)
Comment (PoltoS) [reply Dockerfile]: Le'ts keep it minimalistic and add back if required

File: README.md (hunk 4 above)
Comment (PoltoS) [reply README.md]: I know. Let's use the new one

**Review by PoltoS — APPROVED** [review]

**Pull request merged**